qa/rpc-tests/disablewallet.py

```diff
@@ -29,5 +29,20 @@ def run_test (self):
         x = self.nodes[0].validateaddress('mneYUmWYsuk7kySiURxCi3AGxrAqZxLgPZ')
         assert(x['isvalid'] == True)
 
+        # Checking mining to an address without a wallet
+        try:
+            self.nodes[0].generatetoaddress(1, 'mneYUmWYsuk7kySiURxCi3AGxrAqZxLgPZ')
+        except JSONRPCException,e:
+            assert("Invalid address" not in e.error['message'])
+            assert("ProcessNewBlock, block not accepted" not in e.error['message'])
+            assert("Couldn't create new block" not in e.error['message'])
+
+        errorString = ''
```

Comment:
Please remove this line. The variable is unused and I'd prefer not to use it ever again: see 3b4324b